graphs_sections_A-I.R

```diff
@@ -906,20 +906,20 @@ plot_webchart(filepath="Income and employment/IE-worklessness-chart2", chart_typ
 plot_webchart(filepath="Injuries/inj_road_traffic_chart_1", chart_type = "multiline",
            xvar="class2", yvar="measure", group="class1", pal_col=palnogroups,
            title="Chart 1. Road traffic casualties, Scotland",
-           sourc="<a href='https://www.transport.gov.scot/publication/reported-road-casualties-scotland-2017/'>Transport Scotland</a>", 
-           xaxtitle="Year", yaxtitle="Number")
+           sourc="<a href='https://www.transport.gov.scot/publication/reported-road-casualties-scotland-2019/statistical-tables-1/'>Transport Scotland</a>. Injury severity definitions changed in 2019.", 
+           xaxtitle="Year", yaxtitle="Number of casualties")
 
 plot_webchart(filepath="Injuries/inj_seccare_chart1", chart_type = "multiline",
         xvar="class1", yvar="measure", group="class2", pal_col=palalccondition,
         title="Chart 1. Injury related hospital admissions, Scotland", 
-        sourc="<a href='http://www.isdscotland.org/Health-Topics/Emergency-Care/Unintentional-Injuries/'>SMR01, ISD</a>", 
-        xaxtitle='Year', yaxtitle="Age-sex standarized rate per 100,000")
+        sourc="<a href='https://beta.isdscotland.org/find-publications-and-data/health-services/hospital-care/unintentional-injuries/'>Unintentional Injuries March 2020 Table 3</a>", 
```

Comment:
I would change the source to: "PHS Unintentional Injuries publication Table 3"

Reply:
Would you be happy with "PHS Unintentional Injuries publication March 2020 Table 3"? I can't see a way to link to this specific release of the publication, so I thought it might be important to mention the date? 

(Well, I could link directly to the excel file, but I feel that removes important context.)